dips/models.py

```diff
@@ -437,3 +437,22 @@ class Setting(models.Model):
 
     def __str__(self):
         return self.name
+
+
+class StaticContent(models.Model):
+    key = models.CharField(max_length=50, primary_key=True)
+    content = models.TextField(_("content"), blank=True)
```

Comment:
@jraddaoui , wouldn't be a good idea to have the model mapped out after the concept of a "Page"? E.g. a model where you can also fit the title of the page or in the future things like its permalink? Or you're thinking on something more generic?

Reply:
Good point @sevein! I didn't go that way because some of the contents are added as part of a page, where there are other elements already. There are some follow-ups in the related issue about those "static content" sections.